mypy/test/data/check-overloading.test

```diff
@@ -91,12 +91,6 @@ class A:
   def f(self, x: 'B') -> 'B': pass
 class B: pass
 
-[case testNonConsecutiveOverloads]
```

Comment:
I think what happened here is that long ago mypy allowed function overloading without `@overload`. This test was never updated and neither was the error message! (So it's fine to delete. :-)